content/en/agent/kubernetes/control_plane.md

```diff
@@ -313,7 +313,7 @@ scheduler:
 
 ## Kubernetes on Amazon EKS {#EKS}
```

Comment:
not sure the update from this file should be done in the PR. 
nothing is related to SYN CI/CD.
Better to have a dedicated one for your history